aspnetcore/fundamentals/configuration/index.md

```diff
@@ -345,6 +345,35 @@ When an environment variable is discovered and loaded into configuration with an
 
 <a name="jcp"></a>
 
+## File configuration provider
+
+<xref:Microsoft.Extensions.Configuration.FileConfigurationProvider> is the base class for loading configuration from the file system. The following configuration providers derive from `FileConfigurationProvider`:
+
+* [INI configuration provider](#ini-configuration-provider)
+* [JSON configuration provider](#jcp)
```

Comment:
```suggestion
* [JSON configuration provider](#json-configuration-provider)
```

Reply:
There are plenty more `#jcp` throughout the document, and a `<a name="jcp"></a>` somewhere in a non-logical location.
Shall I change them all to `#json-configuration-provider` and remove the `<a name="jcp"></a>`?

It looks like there are plenty more of these, like `#evcp`, `#fcp`. Just search for `<a name=` to find all of them (and see that some are in a non-logical location).

Maybe a separate PR is more appropiate?

Reply:
> Shall I change them all to #json-configuration-provider and remove the <a name="jcp"></a>?
Yes, please. Make the "#jcp" changes in this PR. The anchor tag approach you pointed out is a legacy approach. The other anchors can stay for now.

Reply:
@scottaddie  the `#jcp` is not the legacy approach. It works better and is easier to update the doc. When 5 is released, we can update the 5.0 moniker section to use `#jcp5` and the links in the 3.x section continue to work. Another advantage is if the heading changes, the link doesn't.